lua/lspconfig/configs/millet.lua

```diff
@@ -17,8 +17,8 @@ To use with nvim:
 
 1. Install a Rust toolchain: https://rustup.rs
 2. Clone the repo
-3. Run `cargo build --release --bin lang-srv`
-4. Move `target/release/lang-srv` to somewhere on your $PATH as `millet`
+3. Run `cargo build --release --bin millet-ls`
+4. Move `target/release/millet-ls` to somewhere on your $PATH as `millet`
```

Comment:
is this the only actual change? the changes to `doc/configs.txt` and `doc/configs.md` should be reverted